imap_processing/idex/idex_constants.py

```diff
@@ -25,3 +26,27 @@ class IdexConstants:
     DATA_MAX: int = 4096
     SAMPLE_RATE_MIN: int = -130
     SAMPLE_RATE_MAX: int = 130
+
+
+# FM sampling rate (quartz oscillator)
+# Seconds per sample.
+FM_SAMPLING_RATE = 0.0038466235767167234e-6
+# Nanoseconds to seconds conversion
+NS_TO_S_CONV_FACTOR = 1e-9
+# Microseconds to seconds conversion
+MS_TO_S_CONV_FACTOR = 1e-6
```

Comment:
Others are leaving off the `CONV_FACTOR` and just calling it `NS_TO_S`. I also wonder if we should put some of these constants into a global constants namespace rather than everyone redefining the same factors... Maybe for later.